doc/include/datamodel_detector.rst

```diff
@@ -0,0 +1,23 @@
+Key                Type       Description                                                                                                                                                                                                                                             
+=================  =========  ========================================================================================================================================================================================================================================================
+``binning``        str        Binning in PypeIt orientation (not the original)                                                                                                                                                                                                        
+``darkcurr``       int,float  Dark current (e-/hour)                                                                                                                                                                                                                                  
+``dataext``        int        Index of fits extension containing data                                                                                                                                                                                                                 
+``datasec``        ndarray    Either the data sections or the header keyword where the valid data sections can be obtained, one per amplifier. If defined explicitly should be in FITS format (e.g., [1:2048,10:4096]).                                                               
+``det``            int        PypeIt designation for detector number (1-based).                                                                                                                                                                                                       
+``gain``           ndarray    Inverse gain (e-/ADU). A list should be provided if a detector contains more than one amplifier.                                                                                                                                                        
+``mincounts``      int,float  Counts in a pixel below this value will be ignored as being unphysical                                                                                                                                                                                  
+``nonlinear``      int,float  Percentage of detector range which is linear (i.e. everything above nonlinear*saturation will be flagged as saturated)                                                                                                                                  
```

Comment:
I think the one `*` is causing troubles.
See: https://pypeit.readthedocs.io/en/imgproc/api/pypeit.images.detector_container.html#pypeit.images.detector_container.DetectorContainer

Reply:
Fixed.